client/my-sites/email/titan-new-mailbox/index.jsx

```diff
@@ -127,6 +140,26 @@ const TitanNewMailbox = ( {
 						/>
 					</FormLabel>
 					{ hasPasswordError && <FormInputValidation text={ passwordError } isError /> }
+					{ hiddenFieldNames.includes( TITAN_PASSWORD_RESET_FIELD ) && ! showAlternateEmail && (
+						<FormSettingExplanation>
+							{ translate( 'Your password reset email is {{strong}}%(primaryEmail)s{{/strong}}.', {
+								args: {
+									primaryEmail,
+								},
+								components: {
+									strong: <strong />,
+								},
+							} ) }
+							<Button
+								primary
+								className="titan-new-mailbox__show-alternate-email"
+								borderless
+								onClick={ showAlternativeEmailField }
+							>
+								{ translate( 'Change it' ) }
+							</Button>
```

Comment:
Because Linter asks me to use a Button instead :(

![image](https://user-images.githubusercontent.com/5689927/159464976-f155c9f5-d8f8-4302-a906-3dc2544ea185.png)

Reply:
I've sorted out adding an empty href